.github/workflows/ci.yml

```diff
@@ -0,0 +1,94 @@
+name: Integration Tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  integration-tests:
+    runs-on: ubuntu-18.04
```

Comment:
Do we need to have one for RHEL as well?

Reply:
it's just the OS which is used to orchestrate the operations, so it doesn't really matter here.
The Cassandra nodes themselves are running on Ubuntu as well, but that's because it's the only OS tlp-cluster can handle (it's not controlled by this setting, but by the AMIs that we use to spin up the instances).
No RHEL/Centos support in tlp-cluster for now.